src/OVAL/probes/SEAP/seap-descriptor.c

```diff
@@ -40,8 +39,8 @@ SEAP_desctable_t *SEAP_desctable_new (void)
         return(t);
 }
 
-int SEAP_desc_add (SEAP_desctable_t *sd_table, SEXP_pstate_t *pstate,
-                         SEAP_scheme_t scheme, void *scheme_data)
+int SEAP_desc_add (SEAP_desctable_t *sd_table, SEAP_scheme_t scheme,
```

Comment:
I think that in future we will also remove SEAP scheme.

Reply:
While I was here, I updated this line to remove the space.